### ISSUE TITLE
Many sat62 cli tests fails because of the below failures

### ISSUE THREAD

**Issue by kbidarkar** [SatelliteQE/robottelo#5866]
NOTE:  
Below are the changes since the last run of Sat6.2.14
1) This is tested with the python3.6 interpreter
2) Also, `cryptography` module was updated to version `2.1.4` from `1.4`, to avoid `cryptography` module installation failures. Just used the sat6.3.z version.
3) The `cryptography` module with version `1.8.2` also seems to install fine.
4) Refer Build 98 of Sat6.2-Tier1 for more logs.

``` 
def test_positive_disable_by_name(self):

"""Disable repo from reposet by names of reposet, org and

product


@id: 1690a701-ae41-4724-bbc6-b0adba5a5319


@expectedresults: Repository was disabled

"""

>       org = make_org()

tests/foreman/cli/test_repository_set.py:269:

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

robottelo/decorators/__init__.py:197: in cacheable_function

new_object = func(options)

robottelo/cli/factory.py:1665: in make_org

return create_object(Org, args, options)

robottelo/cli/factory.py:116: in create_object

result = cli_object.create(options)

robottelo/cli/base.py:158: in create

cls._construct_command(options), output_format='csv')

robottelo/cli/base.py:262: in execute

timeout=timeout,

robottelo/ssh.py:264: in command

return execute_command(cmd, connection, output_format, timeout)

robottelo/ssh.py:277: in execute_command

_, stdout, stderr = connection.exec_command(cmd, timeout)

../../shiningpanda/jobs/02bbb313/virtualenvs/d41d8cd9/lib/python3.6/site-packages/paramiko/client.py:425: in exec_command

stdin = chan.makefile('wb', bufsize)

../../shiningpanda/jobs/02bbb313/virtualenvs/d41d8cd9/lib/python3.6/site-packages/paramiko/channel.py:794: in makefile

return ChannelFile(*([self] + list(params)))

../../shiningpanda/jobs/02bbb313/virtualenvs/d41d8cd9/lib/python3.6/site-packages/paramiko/channel.py:1231: in __init__

self._set_mode(mode, bufsize)

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <paramiko.ChannelFile from <paramiko.Channel 0 (closed) -> <paramiko.Transport at 0x41fe4048 (unconnected)>>>

mode = 'wb', bufsize = None

def _set_mode(self, mode='r', bufsize=-1):

"""

Subclasses call this method to initialize the BufferedFile.

"""

# set bufsize in any event, because it's used for readline().

self._bufsize = self._DEFAULT_BUFSIZE

>       if bufsize < 0:

E TypeError: '<' not supported between instances of 'NoneType' and 'int'

../../shiningpanda/jobs/02bbb313/virtualenvs/d41d8cd9/lib/python3.6/site-packages/paramiko/file.py:460: TypeError

----------------------------- Captured stdout call -----------------------------

2018-03-23 00:43:40 - robottelo - DEBUG - Started Test: RepositorySetTestCase/test_positive_disable_by_name

2018-03-23 00:43:40 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5641fe44a8

2018-03-23 00:43:40 - robottelo.ssh - INFO - Connected to [qe-sat62-rhel7.satellite6.redhat.com]

2018-03-23 00:43:40 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8 hammer -v -u admin -p changeme --output=csv organization create --name="hp3f4X"'

2018-03-23 00:43:40 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5641fe44a8

2018-03-23 00:43:40 - robottelo - DEBUG - Finished Test: RepositorySetTestCase/test_positive_disable_by_name

_______________ RepositorySetTestCase.test_positive_enable_by_id _______________

self = <tests.foreman.cli.test_repository_set.RepositorySetTestCase testMethod=test_positive_enable_by_id>

@tier1
```

**Comment by ldjebran** [comment]
The problem is in this line https://github.com/SatelliteQE/robottelo/blob/6.2.z/robottelo/ssh.py#L277
the line of the code is like this now
```python
_, stdout, stderr = connection.exec_command(cmd, timeout)
```
but the client exec_command is like
```
    def exec_command(
        self,
        command,
        bufsize=-1,
        timeout=None,
        get_pty=False,
        environment=None,
    ):
```python
that's why the buufer is changed as the cmd passed timeout as second arg

we should always pass kwargs as kwargs
the resolution should be like this

```python
_, stdout, stderr = connection.exec_command(cmd, timeout=timeout)
```
this was resolved in master and 6.3.z when we introduced ssh command timeout, but not resolved in 6.2.z

**Comment by kbidarkar** [comment]
Fixed in #5869
